poetry.lock

```diff
@@ -1563,7 +1578,7 @@ url_preview = ["lxml"]
 [metadata]
 lock-version = "1.1"
 python-versions = "^3.7.1"
-content-hash = "c24bbcee7e86dbbe7cdbf49f91a25b310bf21095452641e7440129f59b077f78"
+content-hash = "b3cb4f0789eaa2bbba032e87a173bf7e790db2964f96556ea68d311bfe76715f"
```

Comment:
Not sure what happened here---might be fixed by regenerating the lockfile with poetry 1.1.14

Reply:
Oh it's the hash of pyproject.toml, not of a package!